src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -55,6 +54,14 @@ function formatTemp(temps: string[]): string {
   return temps.filter(Boolean).join("-");
 }
 
+function computeTemperatureStepSize(hass: HomeAssistant, config: Config) {
+  const stateObj = hass.states[config.entity];
+  if (stateObj.attributes.target_temp_step) {
+    return stateObj.attributes.target_temp_step;
+  }
+  return hass.config.unit_system.temperature.indexOf("F") === 1 ? 1 : 0.5;
```

Comment:
We shouldn't do indexOf, we should do `=== UNIT_F` (already defined in `const.ts`)